nemo/collections/asr/models/rnnt_models.py

```diff
@@ -727,6 +727,44 @@ def training_step(self, batch, batch_nb):
 
         return {'loss': loss_value}
 
+    def stream_step(
```

Comment:
Yep this duplication is why this should go into ASRModuleMixin. Switch logic for ctc / rnnt by calling two separate functions (imported from another file) to keep the mixin clean.

Reply:
done.

Reply:
moved it to the ASRModuleMixin.